catalogs/catalog-jdbc-mysql/src/main/java/org/apache/gravitino/catalog/mysql/converter/MysqlTypeConverter.java

```diff
@@ -43,6 +45,12 @@ public class MysqlTypeConverter extends JdbcTypeConverter {
   @Override
   public Type toGravitino(JdbcTypeBean typeBean) {
     switch (typeBean.getTypeName().toLowerCase()) {
+      case BIT:
+      case BIT_UNSIGNED:
+        if (typeBean.getColumnSize() == null || Integer.parseInt(typeBean.getColumnSize()) <= 1) {
```

Comment:
The fact that `getColumnSize()` returns a string is a surprise to me.
Anyway.

Reply:
> The fact that `getColumnSize()` returns a string is a surprise to me. Anyway.

The same to me.

@sunxiaojian could you plz open a new issue to track this?  For changing the `columnSize` type to an integer type.

Reply:
@mchades Okay, this should be of type int

Reply:
thx a lot. Tracked by #5784